exchanges/bybit/bybit_websocket.go

```diff
@@ -63,94 +65,61 @@ var defaultSubscriptions = subscription.List{
 	{Enabled: true, Asset: asset.Spot, Channel: subscription.OrderbookChannel, Levels: 50},
 	{Enabled: true, Asset: asset.Spot, Channel: subscription.AllTradesChannel},
 	{Enabled: true, Asset: asset.Spot, Channel: subscription.CandlesChannel, Interval: kline.OneHour},
-	{Enabled: true, Asset: asset.Spot, Authenticated: true, Channel: subscription.MyOrdersChannel},
-	{Enabled: true, Asset: asset.Spot, Authenticated: true, Channel: subscription.MyWalletChannel},
-	{Enabled: true, Asset: asset.Spot, Authenticated: true, Channel: subscription.MyTradesChannel},
+	// Authenticated channels are currently being managed by the `generateAuthSubscriptions` method for the private connection
+	// TODO: expand subscription template generation to handle authenticated subscriptions across all assets
 }
 
 var subscriptionNames = map[string]string{
 	subscription.TickerChannel:    chanPublicTicker,
 	subscription.OrderbookChannel: chanOrderbook,
 	subscription.AllTradesChannel: chanPublicTrade,
 	subscription.MyOrdersChannel:  chanOrder,
-	subscription.MyTradesChannel:  chanExecution,
 	subscription.MyWalletChannel:  chanWallet,
+	subscription.MyTradesChannel:  chanExecution,
```

Comment:
[ERROR] | WEBSOCKET | 07/08/2025 07:41:11 | Bybit Error subscribing subscribe with request ID 2 msg: Invalid topic :[execution.spot]

I've been getting this using both subscriptions from master and generated ones on this branch